examples/utils/oauth.js

```diff
@@ -1,16 +1,21 @@
 // Authenticate with Genesys Cloud
 function authenticate(client, pcEnvironment) {
+    // Allow targeting a different environment when host app is running locally
+    const platformEnvironment = pcEnvironment === 'localhost' ? 'mypurecloud.com' : pcEnvironment;
     /*
     * Note: To use this app in your own org, you will need to create your own OAuth2 Client(s)
     * in your Genesys Cloud org.  After creating the Implicit grant client, map the client id(s) to
     * the specified region key(s) in the object below, deploy the page, and configure an app to point to that URL.
     */
     const pcOAuthClientIds = {'mypurecloud.com': 'implicit-oauth-client-id-here'};
-    const clientId = pcOAuthClientIds[pcEnvironment];
+    const clientId = pcOAuthClientIds[platformEnvironment];
     if (!clientId) {
         return Promise.reject(new Error(pcEnvironment + ': Unknown/Unsupported Genesys Cloud Environment'));
```

Comment:
nit: you could use platformEnvironment, maybe with an extra note when localhost, here so the dev knows which env localhost was mapped to and that it's not in their oauth client list.

Reply:
ahh good catch 👍